docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>I'd also add a section title above, e.g., <code>Setup and helper functions</code> (after extracting the plotting function as Marco suggests). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>